ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -116,6 +122,100 @@ public ReactScrollView(Context context, @Nullable FpsListener fpsListener) {
     mScroller = getOverScrollerFromParent();
     setOnHierarchyChangeListener(this);
     setScrollBarStyle(SCROLLBARS_OUTSIDE_OVERLAY);
+
+    ViewCompat.setAccessibilityDelegate(
+        this,
+        new AccessibilityDelegateCompat() {
```

Comment:
This delegate appears to be almost identical to the one in ReactHorizontalScrollView.java, so we should probably split this out into its own class and use it in both places, rather then duplicating the code.

I can't think of any reason these would ever likely to have behavior that is different enough that would warrant them both implementing this separately.